### DISCUSSION TITLE
cpecify postgres client connection options?

### DISCUSSION THREAD

**Discussion by Torkolis** [requarks/wiki#6895] [Help / Questions] [answered]
hi, how can I specify client connection options? I have worked with java spring boot a lot and there I can specify those in the jdbc string, for example:
`"jdbc:postgresql://localhost:5432/postgres?options=-c%20search_path=test,public,pg_catalog%20-c%20statement_timeout=90000"`
how would this be possible with wiki js?

**Comment by NGPixel** [comment, marked as answer]
Use env variable `DATABASE_URL`
